content/docs/cdn-links.md

```diff
@@ -1,37 +1,37 @@
 ---
 id: cdn-links
-title: CDN Links
+title: Linki do CDN-ów
 permalink: docs/cdn-links.html
 prev: create-a-new-react-app.html
 next: hello-world.html
 ---
 
-Both React and ReactDOM are available over a CDN.
+Zarówno React, jak i ReactDOM są dostępne przez CDN.
 
 ```html
 <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
 <script crossorigin src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>
 ```
 
-The versions above are only meant for development, and are not suitable for production. Minified and optimized production versions of React are available at:
+Powyższe wersje są przeznaczone tylko do programowania, nie nadają się na środowiska produkcyjne. Zminifikowane i zoptymalizowane wersje produkcyjne Reacta są dostępne pod adresem:
```

Comment:
Myslę nad lepszym tłumaczeniem słowa `for development`, wydaje mi się, że `do programowania` nie oddaje sensu tego zdania. Może moglibyśmy przerobić to zdanie tak, by mówiło po prostu, że nie są to urle do wykorzystania na produkcji?

Reply:
```suggestion
Powyższe wersje przeznaczone są do uruchamiania na serwerach deweloperskich, nie nadają się natomiast na środowiska produkcyjne. Zminifikowane i zoptymalizowane wersje produkcyjne Reacta są dostępne pod adresem:
```